src/pseudopeople/schema_entities.py

```diff
@@ -1,7 +1,9 @@
 from dataclasses import dataclass
 from typing import NamedTuple, Tuple
 
-from pseudopeople.noise_entities import NOISE_TYPES, ColumnNoiseType, RowNoiseType
+from pseudopeople.constants.metadata import Forms
+from pseudopeople.entity_types import ColumnNoiseType, RowNoiseType
+from pseudopeople.noise_entities import NOISE_TYPES
```

Comment:
Were we just importing from the wrong locations?

Reply:
Yep. That didn't solve the circular import (the metadata file solved that), but wasn't what we wanted nonetheless